citadel/indico_citadel/search.py

```diff
@@ -9,7 +9,7 @@
 from requests.exceptions import RequestException
 from werkzeug.urls import url_join
 
-from indico.modules.search.base import IndicoSearchProvider, SearchPlaceholder
+from indico.modules.search.base import IndicoSearchProvider, SearchFilter
```

Comment:
I think the commit for this change in indico itself isn't pushed anywhere

Reply:
I was pending on this PR, but it's there now 👍